05-JWT-Basics/starter/middleware/error-handler.js

```diff
@@ -1,9 +1,12 @@
-const CustomAPIError = require('../errors/custom-error')
+const { CustomAPIError } = require('../../final/errors')
```

Comment:
Same about requesting files from the `final` folder.

```suggestion
const { CustomAPIError } = require('../errors')
```